blackboxopt/utils.py

```diff
@@ -17,7 +17,7 @@
 def get_loss_vector(
     known_objectives: Sequence[Objective],
     reported_objectives: Dict[str, Optional[float]],
-    none_replacement: float = float("NaN"),
+    none_replacement: Union[float, Sequence[float]] = float("NaN"),
```

Comment:
Is there a reason to have a `Sequence[float]` over a `Dict[str, float]`? This would bring it closer to the `reported_objectives` argument, but that's just a minor question.